BUILD

```diff
@@ -216,6 +216,23 @@ go_sdk_tool(
     goroot_relative_path = "bin/gofmt",
 )
 
+alias(
+    name = "goimports",
+    actual = "@org_golang_x_tools//cmd/goimports",
+    visibility = ["//visibility:public"],
+)
+
+genrule(
+    name = "goimports_bin",
+    outs = ["goimports.sh"],
+    cmd = """
+echo '#!/bin/bash' > $@
+echo 'exec $(location @org_golang_x_tools//cmd/goimports) "$$@"' >> $@
+chmod +x $@
+    """,
+    tools = ["@org_golang_x_tools//cmd/goimports"],
+)
```

Comment:
(nit - instead of using `$(location)` does it work if we run `@org_golang_x_tools//cmd/goimports` directly, similar to how we run `@npm//prettier/bin:prettier` which is also an external dep?

Reply:
That absolutely works! Thanks. Lol, `goimports.sh` was failing to find `goimports` when run in CI.

Reply:
Scratch that. What pushed me to writing the `goimports.sh` rule in the first place is that running `bazel run //:goimports` would execute outside the current working directory. I don't know how to get the absolute path of `goimports` without calling $(location ...).